### PR TITLE
[samples] add inference sample for simple snp trainer

### PR DESCRIPTION
**Pull request by o8191**

### CODE REVIEW

File: samples/simple_snp_trainer/cnn_snp_infer.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    """Infer a sample model."""
+    # Create neural factory as per NeMo requirements.
+    nf = nemo.core.NeuralModuleFactory(
+        placement=nemo.core.neural_factory.DeviceType.GPU, checkpoint_dir=parsed_args.model_dir)
+
+    model = create_model()
+
+    # Create test DAG
+    test_dataset = HDFDataLoader(args.test_hdf, batch_size=32,
+                                 shuffle=True, num_workers=args.threads,
```
Comment (tijyojwad) [inline samples/simple_snp_trainer/cnn_snp_infer.py:46]: as I'm thinking about this, I am realizing that we can't use the HDFDataLoader for the inference script here. We'll want to use the ReadPileupDataLoader because that way we'll have access to the Variant entries to serialize outputs to a VCF.

For that one, we also have to set `shuffle=False` so the inference order is the same as the VCF entries order.

**Review by tijyojwad — CHANGES_REQUESTED** [review]
